src/containers/auth/EmailPasswordSignUp.jsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import { Form, Button, Col, Container } from "react-bootstrap";
+import { Redirect } from "react-router-dom";
+import { connect } from "react-redux";
+import { signUp } from "../../actions/authActions";
+
+const CreateEmailPassword = (props) => {
+  const [firstName, setFirstName] = React.useState("");
+  const [lastName, setLastName] = React.useState("");
+  const [email, setEmail] = React.useState("");
+  const [password, setPassword] = React.useState("");
+  const { auth, authError, errMessage } = props;
+  // Redirect to home page after it's signed up and logged in
+  if (auth.uid) {
+    return <Redirect to="/" />;
+  }
+  // Create a new account with "Email & Password" method
+  const handleSubmit = (e) => {
+    const initials = firstName[0] + " " + lastName[0];
+    props.signUp({
+      firstName,
+      lastName,
+      email,
+      password,
+      initials,
+      imageURL: null,
+    });
+    e.preventDefault();
+  };
+  return (
+    <Container>
+      <h3>Creating a new account</h3>
+      <Form>
+        <Form.Row>
+          <Form.Group as={Col} controlId="fisrtName">
+            <Form.Label>First Name:</Form.Label>
+            <Form.Control
+              type="text"
```

Comment:
see other comments about collecting both first and last name

fix place holders

`First name`